pkg/gitops/flux/installer.go

```diff
@@ -283,7 +284,7 @@ func getFluxManifests(opts *api.Git, cs kubeclient.Interface) (map[string][]byte
 		GitLabel:           opts.Operator.Label,
 		GitUser:            opts.Repo.User,
 		GitEmail:           opts.Repo.Email,
-		GitReadOnly:        false,
+		GitReadOnly:        opts.Operator.ReadOnly,
 		Namespace:          opts.Operator.Namespace,
 		ManifestGeneration: true,
 		AdditionalFluxArgs: []string{"--sync-garbage-collection"},
```

Comment:
When using readonly, it would be best to deploy Flux without memcached, this can be done by setting an additional flag `--registry-disable-scanning`.

Reply:
Sounds great! Added in 8e6afca3